packages/cli/src/lib/cmds/init.ts

```diff
@@ -36,10 +36,10 @@ async function confirmTwaConfig(twaManifest: TwaManifest, prompt: Prompt): Promi
   prompt.printMessage(messages.messageWebAppDetailsDesc);
 
   twaManifest.host = await prompt.promptInput(
-      messages.promptHostMessage, twaManifest.host, validateHost);
+      messages.promptHostMessage, twaManifest.host, await validateHost);
```

Comment:
This `await` is not needed as the function is not being invoked, just passed as parameter. Please, check others throughout the code.

```suggestion
      messages.promptHostMessage, twaManifest.host, validateHost);
```